doc/api/repl.md

```diff
@@ -591,11 +595,13 @@ changes:
   * `completer` {Function} An optional function used for custom Tab auto
      completion. See [`readline.InterfaceCompleter`][] for an example.
   * `replMode` {symbol} A flag that specifies whether the default evaluator
-    executes all JavaScript commands in strict mode or default (sloppy) mode.
+    executes all JavaScript commands in strict mode or sloppy mode.
     Acceptable values are:
     * `repl.REPL_MODE_SLOPPY` to evaluate expressions in sloppy mode.
     * `repl.REPL_MODE_STRICT` to evaluate expressions in strict mode. This is
-      equivalent to prefacing every repl statement with `'use strict'`.
+      equivalent to prefacing every repl statement with `'use strict'` or to
+      start the node executable with the `--use-strict` flag. **Default:**
```

Comment:
Can you remove the `--use-strict` check from repl instead of documenting this?

Reply:
I am fine to remove the documentation for it again but I would not remove the check for it in the REPL. It's an implicit behavior that would apply if it's supported by V8 and would not have any effect otherwise.

Reply:
Even though it does seem like people use it frequently and it does seem like it's fine to use in the REPL (I would recommend against using it anywhere else).

Reply:
I removed the documentation again.